Description: No install tests folder
Author: Marcelo Jorge Vieira <metal@debian.org>
Bug-Debian: https://bugs.debian.org/788840

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- pyvows-2.0.6.orig/setup.py
+++ pyvows-2.0.6/setup.py
@@ -56,7 +56,7 @@ setup(
     extras_require={
         'tests': _test_requires,
     },
-    packages=find_packages(),
+    packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
     package_dir={'pyvows': 'pyvows'},
     entry_points={
         'console_scripts': [
